src/utils/hooks/useBootSourceDefaultStorageClass.ts

```diff
@@ -0,0 +1,21 @@
+import { IoK8sApiStorageV1StorageClass } from '@kubevirt-ui/kubevirt-api/kubernetes/models';
+import useDefaultStorageClass from '@kubevirt-utils/hooks/useDefaultStorage/useDefaultStorageClass';
+import { TemplateBootSource } from '@kubevirt-utils/resources/template/hooks/useVmTemplateSource/utils';
+
+type UseBootSourceDefaultStorageClass = (bootSource?: TemplateBootSource) => {
+  isBootSourceSCisDefaultSC: boolean;
```

Comment:
this name looks a bit strange to me, maybe `bootSourceDefaultSC` wdyt?

Reply:
changed it a bit :)